### PR TITLE
Mixer upgrade and clean up post migration

### PR DESCRIPTION
**Pull request by shifucun**
* Upgrade Golang to 1.16 and update libraries
* Move golden response files out of "staging" since there is no "prod" version now.
* Remove references to "datcom-store-dev"
* Remove cache key for PopObs related cache

### CODE REVIEW

**Review by beets — APPROVED** [review]
thx for the cleanup!

File: internal/util/util.go [hunk 1]
```diff
@@ -36,26 +36,16 @@ const (
 	BtPlaceStatsVarPrefix = "d/0/"
 	// BtStatVarGroup is the key for statvar group cache.
 	BtStatVarGroup = "d/1"
-	// BtPopObsPrefix for internal pop obs cache.
-	BtPopObsPrefix = "d/2/"
```
Comment (shifucun) [inline internal/util/util.go:40]: @pradh We could also remove these cache from prophet

Comment (pradh) [reply internal/util/util.go:40]: d/2/ is needed for pop-obs validation pipeline during import, until we switch import flows

so we're not using this obsid->{observation-about, variable-measured} cache?

Comment (shifucun) [reply internal/util/util.go:40]: Not any more, observation properties are fetched using BQ

**Pull request merged**